### PR TITLE
add message definition

### PR DESCRIPTION
**Pull request by zhenglaizhang**

### CODE REVIEW

File: libraries/swagger-validation-common/src/types/event.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+type Stage = "Queued" | "InProgress" | "Completed";
```
Comment (ruowan) [inline libraries/swagger-validation-common/src/types/event.ts]: Where is the stage used?

Comment (zhenglaizhang) [reply libraries/swagger-validation-common/src/types/event.ts]: Oh, sorry, this is not finished yet, I added the file by mistake, removing this in this pr

Comment (zhenglaizhang) [reply libraries/swagger-validation-common/src/types/event.ts]: @ruowan  removed, will sent out another pr including events

**Comment by zhenglaizhang** [comment]
@ruowan @NullMDR  please help review this, thx.

**Review by ruowan — APPROVED** [review]

**Pull request merged**